docs/conf.py

```diff
@@ -24,6 +24,9 @@
 import django
 from django.utils import six
 
+sys.path.insert(0, os.path.join(os.path.abspath(os.path.dirname(__file__)), '..'))
```

Comment:
Ick. I don't understand why this is needed here, but a similar line is commented-out in the coverage.py repo, and it still works.